app/src/main/AndroidManifest.xml

```diff
@@ -16,7 +16,8 @@
         android:theme="@style/AppTheme">
         <activity
             android:name=".MainActivity"
-            android:windowSoftInputMode="adjustPan">
+            android:windowSoftInputMode="adjustPan"
+            android:configChanges="orientation">
```

Comment:
Rather, why don't you try to eliminate the root cause

Reply:
Shouldn't we use this so that activity is not recreated on rotating the screen?

Reply:
No, what if we use different layouts for portrait and landscape. This is a workaround, not a fix

Reply:
Okay, I'll work on it.